R/train.R

```diff
@@ -55,7 +55,48 @@
 #' @param best_model_number Number best models to be chosen as element of the return.
 #' All trained models will be returned as different element of the return.
 #'
-#' @return A list of all necessary objects for other functions.
+#' @return A list of all necessary objects for other functions. It contains:
+#' `type` The type of the ML task. If the user did not specify a type in the
+#' input parameters, the algorithm recognizes, uses and returns the same type.
+#' It could be 'regression' or 'classification'.
+#' `deleted_columns` Column names from the original data frame that have been
+#' removed in the data preprocessing process, e.g. due to too much correlation
+#' with other columns.
+#' `preprocessed_dataThe` data frame after the preprocessing process - that
+#' means: removing columns with one value for all rows, binarizing the target
+#' column, managing missing values, deleting correlated values, deleting columns
+#' that are ID-like columns and performing Boruta algorithm for selecting most

```

Comment:
Ale my Borutę robimy tylko w extended preprocessing

Reply:
To wyszczególnię, że chodzi o advanced preprocessing.